packages/@okta/vuepress-site/guides/sign-into-mobile-app/check-on-startup/ios/checkfortoken.md

```diff
@@ -37,16 +37,3 @@ authStateManager?.introspect(token: accessToken, callback: { payload, error in
     // handle token valid case
 })
 ```
```

Comment:
@IldarAbdullin-okta Does the developer always need to use the introspect approach or is it optional? What do you recommend? We can move any optional approaches into the library's Github readme.

Reply:
Introspect approach is absolutely optional and looks for me more as a troubleshooting approach. I will remove that section